### ISSUE TITLE
[Question] How do I run an exhaustive sweep job?

### ISSUE THREAD

**Issue by pievalentin** [Azure/azure-sdk-for-python#26958]
- **Package Name**:  azure.ai.ml.sweep
- **Package Version**:  1.0.0
- **Operating System**: Ubuntu
- **Python Version**: py38

Here is the config I have for my sweep job

```python
# define the command
command_job = command(
    code=".",
    command="python ./main.py train --decision_date ${{inputs.decision_date}} --period ${{inputs.period}} --config_path ${{inputs.config_path}}",
    environment="AzureML-lightgbm-3.2-ubuntu18.04-py37-cpu@latest",
    inputs={
        "decision_date": '2014-12-31',
        "period": 12,
        "config_path": "./conf/conf_model.json",
    },
    compute="dstg-drgnfr-D2V3-cc",
)

command_job_for_sweep  = command_job(decision_date=Choice(["2014-12-31"]), period=Choice(range(12,24)))

# apply the sweep parameter to obtain the sweep_job
sweep_job = command_job_for_sweep.sweep(
    compute="dstg-drgnfr-D2V3-cc",
    sampling_algorithm="grid", #NOTE: I was expecting grid to be exhaustive
    primary_metric="None", # NOTE: I use none because I am interested in all the run of the sweep. I don't need to perform a selection
    goal="Minimize",
)

# define the limits for this sweep
sweep_job.set_limits(max_total_trials=20, max_concurrent_trials=10, timeout=7200)

#Submit the job
returned_sweep_job = ml_client.create_or_update(sweep_job)
```
This get submitted successfully. Thing is the trials that AzML generate are not exhaustive. period does not go up to 23 even though I submitted a `range(12,24)`

TL;DR How do I perform a complete grid search?

**Comment by pievalentin** [comment]
Ok my bad it was an easy fix. The grid option do an exhaustive search. Just make sure that here
`sweep_job.set_limits(max_total_trials=20, max_concurrent_trials=10, timeout=7200)`
You specify a `max_total_trials` that encompass all the combination.
